src/locales/hu.yaml

```diff
@@ -1,508 +1,499 @@
+          a Z-eltolás.
+        apply_z_offset_probe: >-
+          A Probe/Bltouch Z-eltolás kiszámítása és frissítése az aktuális értékből
+          történik.
+    table:
+      header:
+        actions: Parancsok
+        end_time: Befejezett
+        estimated_time: Becsült idő
+        filament: Nyomtatószál
+        filament_used: Használt nyomtatószál
+        filament_weight_total: Nyomtatószál súlya
+        first_layer_bed_temp: Első réteg ágy hőmérséklet
+        first_layer_extr_temp: Első réteg extruder hőmérséklet
+        first_layer_height: Első réteg magassága
+        height: Magasság
+        last_printed: Utoljára nyomtatott
+        layer_height: Rétegmagasság
+        modified: Módosított
+        name: Név
+        print_duration: Nyomtatás időtartama
+        size: Méret
+        slicer: Szeletelő
+        slicer_version: Szeletelő verziója
+        start_time: Elindult
+        status: Állapot
+        total_duration: Teljes időtartam
+    title:
+      add_printer: Nyomtató hozzáadása
+      bedmesh: Ágyháló
+      bedmesh_controls: Ágyhálós vezérlők
+      camera: Kamera
+      config_files: Konfigurációs fájlok
+      configure: Konfiguráció
+      console: Konzol
+      endstops: Végállások
+      fans_outputs: Ventilátorok & Kimenetek
+      gcode_preview: G-Kód előnézet
+      history: Történelem
+      home: Otthon
+      jobs: Feladatok
+      limits: Nyomtatókorlátok
+      macros: Makrók
+      other_files: Egyéb fájlok
+      retract: Firmware visszahúzás
+      runout_sensors: Kifutás érzékelők
+      settings: Beállítások
+      stats: Nyomtató statisztikák
+      system: Rendszer
+      system_overview: Rendszer információ
+      temperature: Hőmérsékletek
+      tool: Eszköz
+      tune: Hangolás
+    tooltip:
+      estop: VészSTOP
+      reload_klipper: Újratölti a klipper konfigurációját.
+      reload_restart_klipper: Újratölti a klipper konfigurációját és újraindítja az MCU-kat.
+      restart_klipper: Újraindítja a klipper rendszerszolgáltatást.
+  history:
+    msg:
+      confirm_jobs: Biztos vagy benne? Ezzel minden feladat törlődik.
+      confirm_stats: Biztos vagy benne? Ezzel az összes statisztika törlődik.
+  printer:
+    state:
+      busy: Elfoglalt
+      complete: Befejez
+      idle: Tétlen
+      loading: Betöltés
+      paused: Szüneteltetve
+      printing: Nyomtatás
+      ready: Kész
+      standby: Készenlét
+  setting:
+    btn:
+      add_camera: Kamera hozzáadása
+      add_category: Kategória hozzáadása
+      add_filter: Szűrő hozzáadása
+      add_thermal_preset: Előbeállítás hozzáadása
+      add_user: Felhasználó hozzáadása
+      reset: Újraindítás
+      select_theme: Válassz témát
+    camera_type_options:
+      mjpegadaptive: MJPEG Adaptív
+      mjpegstream: MJPEG Sugárzás
+      video: IP Kamera
+      iframe: HTTP oldal
+    camera_rotate_options:
+      none: Nincs
+      90: 90°
+      180: 180°
+      270: 270°

```

Comment:
The keys need to be surrounded by single quotes `'`, just as it was before the change!